guide/how-it-works/ecash/overview.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: guide
+title: Overview
+description: A primer on ecash on bitcoin
+nav_order: 1
+parent: Ecash
+grand_parent: How it works
+permalink: /guide/how-it-works/ecash/overview
+redirect_from:
+ - /guide/ecash/overview/
+main_classes: -no-top-padding -ecash-page
+image: https://bitcoin.design/assets/images/guide/how-it-works/ecash/ecash.jpg
+---
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/ecash.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/ecash@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/ecash-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/ecash-mobile@2x.jpg"
+   alt-text = "ecash overview illustration"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+<!--
+
+Editor's notes
+
+Figma file: https://www.figma.com/file/ZwRT4mZZ4UNGtsfLz22m8M/Liquidity?node-id=0%3A1
+Figma file for channel reserve UI's: https://www.figma.com/file/6iJpftEbajA3y1ylXQxsrl/Channel-reserve
+
+-->
+
+# Ecash
+{:.no_toc}
+
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
+{:toc}
+</div>
+
+---
+## What is ecash?
+Ecash is a digital payment system that uses cryptographic techniques to ensure secure and private transactions. Developed by David Chaum in 1983, it operates through a mint that issues digital tokens in exchange for deposited funds. These tokens can be spent and received without revealing user identities or transaction details, thanks to blinded signatures. Users can redeem their tokens for the original funds at any time. Since no accounts or personal information are needed; the mint cannot see your balance, tokens, or transaction details, offering a highly private and efficient method for digital payments.
+
+## How ecash on bitcoin works
```

Comment:
> ecash on bitcoin

It is not clear what it means that ecash is "on" bitcoin? The reverse might even be more accurate "bitcoin on ecash", since ecash is a standalone system that can support deposits of bitcoin (and really any other currency or asset).

Reply:
This is good point. I think "bitcoin-backed ecash" might be more accurate. It describe the nature of the ecash more. What do you think of replaced "bitcoin on ecash" with "bitcoin-backed ecash"?

Reply:
That sounds good.

Reply:
I think bitcoin backed or bitcoin denominated both work. I went with 'bitcoin backed' because I think it's more natural sounding but not a hill I will die on. I'm fine with updating it. If you guys feel strongly about bitcoin denominated instead of bitcoin backed then I can update it.